### DISCUSSION TITLE
A way to tweak how tests URLs are generated

### DISCUSSION THREAD

**Discussion by mateuszwojnarowski** [schemathesis/schemathesis#2290] [Q&A]
Hi,

I'm running Schemathesis tests against API Gateway build on Ocelot. 
There's a failure happening every now and then when Schemathesis sends the request with two '*' in the address, f.e. Https://api/v1/api/**. The problem is, Ocelot throws due to regex exception because of two asterisks next to each other.

Is there a way to tune the URL paths the Schemathesis generates, to prevent such scenarios?

**Comment by Stranger6667** [comment]
Hi @mateuszwojnarowski 

You could try the `filter_path_parameters` [hook](https://schemathesis.readthedocs.io/en/stable/extending.html#filtering-data):

```python
@schemathesis.hook
def filter_path_parameters(context, path_parameters):
    return not any("**" in param for param in path_parameters)
```
